app/views/projects/index.html.slim

```diff
@@ -7,6 +7,10 @@ h1.header
   = icon('users')
   span Projects
 
+nav.past-projects
+  - project_years.each do |filter|
+    = link_to filter.titleize, projects_path(filter: filter), class: "label label-default"
```

Comment:
Using these labels is a nice idea 💡 I'd even suggest to align the `teams` page to this pattern and remove the dropdown there in favor of these labels _(would also help us spare some untested JS code there)_